attachment_base_synchronize/models/attachment.py

```diff
@@ -56,8 +57,7 @@ def _compute_hash(self):
     @api.model
     def run_attachment_metadata_scheduler(self, domain=None):
         if domain is None:
-            domain = []
-        domain.append(('state', '=', 'pending'))
+            domain = [('state', '=', 'pending')]
```

Comment:
Did you really want to change the behaviour here ?
The `('state', '=', 'pending')` tuple is now only added in the domain when the domain was `None`, instead of being always added like before.

A unit test calling this method with a domain in arguments would be fine.